higher_health/templates/login.html

```diff
@@ -20,6 +21,12 @@ <h4>Thank you for coming forward and for contributing to the
               health of all citizens AND stopping the spread of the virus.🦠</h4>
           </div>
 
+          <div class="columns-child columns-child--excerpts">
+              <a class="lifebouy-ad" href="https://www.lifebuoy.co.za">
+                <img class="lifebouy-ad"  src="{% static 'images/POST_small.jpg' %}" alt="Higherhealth Logo" role="img">
```

Comment:
I see that there's no `images/POST_small.jpg` in this PR, only, `POST_1.jpg`. 

And then I think we should change the alt text to something more descriptive, like `Lifebouy Advertisement`

Reply:
hmm, my changes don't seem to be coming through

Reply:
@rudigiesler updated